docs/master/api-reference/directives.md

```diff
@@ -1111,6 +1111,47 @@ class PlacedOrder
 }
 ```
 
+## @extends
+
+```graphql
+"""
+This allows types to extends other Types
+
+it inherits properties from the parent type. 
+It overides same attributes present in the parent type.
+'""
+"""
+directive @extends(
+    """
+    The parent Type
+    """
+    parent: String!
+) on OBJECT
```

Comment:
What about multiple inheritance? I don't see why it shouldn't be allowed.

Reply:
@yondifon you keep marking this as resolved, yet have not addressed this comment at all.

Reply:
I think you are looking at the wrong commit. It says (outdated) which I have updated. 


Here's the latest that addresses this. (https://github.com/nuwave/lighthouse/pull/1895/commits/952a3706523b8af1068af9bec9e3655dd42f0037)

Reply:
How does this address multiple inheritance? Do you have a test for it? The directive is missing the `repeatable` keyword as of now.